horizomer/benchmark/parse_output.py

```diff
@@ -241,16 +241,16 @@ def parse_genemark(input_f, genbank_fp):
     atypical_genes = []
     reading = False
     for line in input_f:
-        l = line.strip().split()
-        if len(l) == 2 and l == ['#', 'Length']:
+        x = line.strip().split()
+        if len(x) == 2 and x == ['#', 'Length']:
             reading = True
         # atypical genes have class '2' in the 6th column
-        elif reading and len(l) == 6 and l[5] == '2':
-            (start, end, strand) = (int(l[2].lstrip('<>')),
-                                    int(l[3].lstrip('<>')),
-                                    l[1])
-            for (gene, l) in genes.items():
-                if l[0] == start and l[1] == end and l[2] == strand:
+        elif reading and len(x) == 6 and x[5] == '2':
+            (start, end, strand) = (int(x[2].lstrip('<>')),
+                                    int(x[3].lstrip('<>')),
+                                    x[1])
+            for (gene, x) in genes.items():
+                if x[0] == start and x[1] == end and x[2] == strand:
```

Comment:
Preferably, there would be tests for the parsers.  We can also defer this in an issue

Reply:
There are tests for the parsers, see (here)[https://github.com/biocore/horizomer/blob/master/horizomer/benchmark/tests/test_parse_output.py]. Those `x`s are here because new PEP8 no longer supports `l` being a legit variable name.